### PR TITLE
Replace cargo-make with a standard Makefile

### PR DESCRIPTION
**Pull request by rjoleary**
Signed-off-by: Ryan O'Leary <ryanoleary@google.com>

### CODE REVIEW

**Comment by gkelly** [comment]
https://github.com/oreboot/oreboot/blob/master/README.md#ground-rules has language that needs an update with this.

File: src/mainboard/Makefile.common [hunk 1]
```diff
@@ -0,0 +1,67 @@
+# This Makefile is included from src/mainboards/VENDOR/BOARD/Makefile
```
Comment (rminnich) [inline src/mainboard/Makefile.common]: This Makefile is included from src/mainboards/VENDOR/BOARD/Makefile
                                             in?
                                             by?

Can you set OREBOOT in here too? 
OREBOOT=$(magic)

**Comment by rjoleary** [comment]
> https://github.com/oreboot/oreboot/blob/master/README.md#ground-rules has language that needs an update with this.

Thanks for bringing this up. I will leave it up to rminnich@ to determine if this is allowable (would require changes to ground rules). This attempts to fix issues in #218. Perhaps an entirely different build system is needed.

File: src/mainboard/Makefile.common (hunk 1 above)
Comment (rjoleary) [reply src/mainboard/Makefile.common]: I don't understand. For example "src/mainboard/Makefile.common is included from src/mainboards/sifive/hifive/Makefile".

What is the purpose of the OREBOOT magic?

File: src/mainboard/ast/ast25x0/build.rs [hunk 2]
```diff
@@ -0,0 +1,3 @@
+fn main() {
```
[the remark this answers is not in the document]
Comment (rjoleary) [reply src/mainboard/ast/ast25x0/build.rs]: Without this, it seems cargo doesn't realize when the linker script changes and won't relink the project.

**Pull request merged**